src/assignmentone/Validator.java

```diff
@@ -0,0 +1,115 @@
+package assignmentone;
+
+import assignmentone.exceptions.InvalidInputError;
+
+import java.util.List;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.ArrayList;
+
+/*
+ * This class is use to check the inputs entered by user
+ */
+public class Validator {
+
+    /*
+     * This method check validity of type and name
+     *
+     * @param args list of arguments
+     */
+    private void nameAndTypeCheck(final List<String> args) {
+        final var name = Constants.dash.concat(Constants.name);
+        final var type = Constants.dash.concat(Constants.type);
+        if (args.indexOf(name) != Constants.ZERO) {
+            throw new InvalidInputError("Input should start with -name");
```

Comment:
make a error constant file for all error messages

Reply:
made